service_management/test/resources/logback-test.xml

```diff
@@ -13,7 +13,7 @@
 
   <logger name="play" level="WARN"/>
   <logger name="akka" level="ERROR"/>
-  <logger name="slick.jdbc" level="WARN"/>
+  <logger name="slick.jdbc" level="DEBUG"/>
```

Comment:
Please change back to `WARN` before committing and pushing code.
It's a good idea to get used to looking over each file you want to commit and adding them explicitly, instead of doing a `git add .` from the root of the project.